lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -212,6 +212,7 @@ def initialize(options)
       @devices_list.flatten!
       @devices_list.compact!
       load_devices
+      @image_path = @sm.boot_image_path
```

Comment:
This is different from what the VM actually uses if `create_snapshot`.

Reply:
`@image_path` can be simply omitted from `dump_config`. I think that information is not more useful than other omitted information.

Reply:
The variable was updated to @sm.boot_snapshot_path, which is precisely the path we require.

Reply:
> The variable was updated to @sm.boot_snapshot_path, which is precisely the path we require.

It makes the printed path wrong when `create_snapshot` is false.